teamcity/datadog_checks/teamcity/common.py

```diff
@@ -0,0 +1,140 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import re
+import time
+from copy import deepcopy
+
+import requests
+
+from datadog_checks.base import ConfigurationError
+
+from .constants import BUILD_EVENT, DEPLOYMENT_EVENT, EVENT_STATUS_MAP, RESOURCE_URL_MAP
+
+
+def construct_build_configs_filter(check):
+    """
+    Construct build configuration filter
+    """
+    excluded_build_configs = set()
+    included_build_configs = set()
+    for project in check.monitored_build_configs:
```

Comment:
You can use `.items()` here instead to get a list of `monitored_build_configs`'s (key, value) tuple pairs